templates/test/infopanel.html.ep

```diff
@@ -10,7 +10,11 @@
         % }
         <div class="panel <%= $panelclass %>" id="info_box">
             <div class="panel-heading">
-                Results for <%= $job->name %>
+                Results for
+                % if (current_route 'latest') {
+                    <%= link_to $job->id => url_for ('test', testid => $job->id) %>:
+                % }
+                <%= $job->name %>
```

Comment:
please don't use this syntax. If it's the only thing in a line just use 

```
%= $job->name
```

But no need to change now - just remember